integrations/jira/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,35 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: project
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .key | tostring
+          title: .name | tostring
```

Comment:
What happens if those are null?
Maybe if its null we should pass null instead of "null"

Reply:
`key` has to exist otherwise you have no Jira object (no matter the type)
I'll remove the `tostring` on `.name`